rm_vk/rm_vk/AllGroupsTableViewController/AllGroupsTableViewController.swift

```diff
@@ -0,0 +1,76 @@
+// AllGroupsTableViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Экран общего списка групп
+final class AllGroupsTableViewController: UITableViewController {
+    // MARK: - Constants
+
+    private enum Constants {
+        static let cellIdentifier = "AllGroupTableViewCell"
+        static let oneGroupImageName = "liza"
+        static let twoGroupImageName = "maggie"
+        static let oneGroupTitle = "Liza's club"
+        static let twoGroupTitle = "Agu"
+        static let oneGroupDescription = "Отличники"
+        static let threeGroupImageName = "bart"
+        static let threeGroupTitle = "Caramba"
+    }
+
+    // MARK: - Public Properties
+
+    var groups = [
+        Group(
+            avatarImageName: Constants.oneGroupImageName,
+            title: Constants.oneGroupTitle,
+            description: Constants.oneGroupDescription
+        ),
+        Group(
+            avatarImageName: Constants.twoGroupImageName,
+            title: Constants.twoGroupTitle,
+            description: nil
+        ),
+        Group(
+            avatarImageName: Constants.threeGroupImageName,
+            title: Constants.threeGroupTitle,
+            description: nil
+        ),
+        Group(
+            avatarImageName: Constants.oneGroupImageName,
+            title: Constants.oneGroupTitle,
+            description: Constants.oneGroupDescription
+        ),
+        Group(
+            avatarImageName: Constants.twoGroupImageName,
+            title: Constants.twoGroupTitle,
+            description: nil
+        ),
+        Group(
+            avatarImageName: Constants.threeGroupImageName,
+            title: Constants.threeGroupTitle,
+            description: nil
+        )
+    ]
+
+    // MARK: - Lifecycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+
+    // MARK: - UITableViewDataSource
```

Comment:
```suggestion
    // MARK: - UITableViewDataSource
```
речь шла о // MARK: - Public methods , но может и ничего не скажут